test_runner/src/main/kotlin/ftl/adapter/google/ListLocales.kt

```diff
@@ -0,0 +1,59 @@
+package ftl.adapter.google
+
+import ftl.api.Locale
+import ftl.environment.TAGS
+import ftl.environment.TestEnvironmentInfo
+import ftl.environment.createTableColumnFor
+import ftl.environment.getOrCreateList
+import ftl.environment.tagToSystemOutColorMapper
+import ftl.run.exception.FlankGeneralError
+import ftl.util.applyColorsUsing
+import ftl.util.buildTable
+
+fun List<Locale>.asPrintableTable() = createTestEnvironment().createLocalesTable()
+
+fun List<Locale>.createTestEnvironment() =
+    fold(mutableMapOf<String, MutableList<String>>()) { allLocales, locale ->
+        allLocales.apply {
+            getOrCreateList(LOCALE).add(locale.id.orEmpty())
+            getOrCreateList(NAME).add(locale.name.orEmpty())
+            getOrCreateList(REGION).add(locale.region.orEmpty())
```

Comment:
This file is not part of `api-adapter-client`. It's rather the part of the presentation layer but for this stage of refactoring it should be left where it was previously. I mean the whole file, not just the selected part.

Reply:
👍

Reply:
updated.